dashboard/client/src/api/functionsApi.js

```diff
@@ -177,6 +177,18 @@ class FunctionsApi {
       return res.data;
     });
   }
+
+  fetchFunctionLog({
+                  longFnName,
+                  user
+                }) {
+    const url = `${
+        this.apiBaseUrl
+    }/function-logs?function=${longFnName}&user=${user}`;
+    return axios.get(url).then(res => {
```

Comment:
Any need for `catch(err => { //etc })`?

Reply:
Good point, probably want it higher up, so the component decided what to do if theres an error (status code non 2XX)